Cargo.toml

```diff
@@ -10,12 +10,16 @@ categories = ["command-line-utilities", "text-processing"]
 edition = "2018"
 
 [dependencies]
+ahash = "0.3"
 anyhow = "1"
-err-derive = "0.2.4"
+err-derive = "0.2"
```

Comment:
Why did you change the err-derive version here?

Reply:
I think that any 0.2 API should do. There's no need to be overly specific.

Reply:
I don't think this helps; please revert the change.

For versions whose major version is 0, Cargo treats the minor version as the major version (that is, minor version bumps are semver-incompatible). In turn, that means that both bug fixes and features go in micro version bumps, so it's never guaranteed that 0.2.0 will work if you've only tested with 0.2.4. As such, since 0.2.0 and 0.2.4 are semver-compatible, Cargo will never use anything below 0.2.4 for new projects if 0.2.4 once 0.2.4 has been released (except in the case that another crate in the same dependency graph specifies a precise dependency like `=0.2.2` -- which I think is exceedingly rare).

Reply:
Ok.